Discuss and review changes in this comparison with others.

orangebud opened the pull request:
…ethods from StepDefinition class were removed.

orangebud committed:
Small modifications. Scenario outline has been created, unnecessary methods from StepDefinition class were removed.

v-rgasztany commented:
Please add some logging mechanism
http://javarevisited.blogspot.ie/2013/08/why-use-sl4j-over-log4j-for-logging-in.html

orangebud committed:
Generated text file has been removed.
Duplicated method has been fixed.
1 screenshot taken instead of 2.
Now using log4j for logging.

v-rgasztany commented on pom.xml:
```diff
@@ -46,6 +46,38 @@
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-log4j12</artifactId>
+            <version>1.6.1</version>
```
please choose the latest verion
http://mvnrepository.com/artifact/org.slf4j/slf4j-log4j12/1.7.12

v-rgasztany commented on pom.xml:
```diff
@@ -46,6 +46,38 @@
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>1.6.1</version>
```
same

orangebud committed:
Log4J has been configured properly
Dependencies added to pom.xml

v-rgasztany commented on pom.xml:
```diff
@@ -46,6 +46,37 @@
+        <dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+            <version>1.2.17</version>
```
pls update

v-rgasztany commented on pom.xml:
```diff
@@ -46,6 +46,37 @@
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
+            <version>1.2.17</version>
```
pls update

v-rgasztany commented on pom.xml:
```diff
@@ -46,6 +46,37 @@
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-api</artifactId>
+            <version>2.1</version>
```
pls update

orangebud committed:
Refreshed dependencies for the latest versions in the pom.xml file

orangebud committed:
Modified log mechanism
